src/Data/Sequence/Circular.hs

```diff
@@ -1,16 +1,11 @@
 {--
- This module models a Circular Buffer using Data.Sequence's `Seq` as foundation.
+ This module models a circular buffer using a vector as a foundation.
```

Comment:
It would be great to have tests for this module too so that we don’t need rely on the block cache tests.

Reply:
I can surely do that (I think that's a good idea anyway).